tools-local/unix/search-tool.sh

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/env bash
+
+usage()
+{
+    echo "usage: $0 <repository-root> <tool-name> <override-scripts-folder-path> [strict-tool-version-match]"
```

Comment:
Yes, it is required. Updated as -
echo "usage: $0 <repository-root> <tool-name> <override-scripts-folder-path> <strict-tool-version-match>"

---
In reply to: [110420064](https://github.com/ravimeda/corefx/pull/9#discussion_r110420064) [](ancestors = 110420064)